packages/@aws-cdk/aws-accessanalyzer/package.json

```diff
@@ -62,23 +62,7 @@
     "url": "https://aws.amazon.com",
     "organization": true
   },
-  "jest": {
-    "moduleFileExtensions": [
-      "js"
-    ],
-    "coverageThreshold": {
-      "global": {
-        "branches": 60,
```

Comment:
When unifying, do we still have support for separate coverage % for individual modules?

Reply:
Yes! And it can be specified here. All of the top-level properties under 'jest' can be overridden in package.json.